src/EditorFeatures/CSharp/CSharpEditorFeatures.csproj

```diff
@@ -150,6 +150,9 @@
     <Compile Include="InlineRename\CSharpEditorInlineRenameService.cs" />
     <Compile Include="LanguageServices\CSharpContentTypeLanguageService.cs" />
     <Compile Include="LineSeparators\CSharpLineSeparatorService.cs" />
+    <Compile Include="NamedArguments\CodeFixes\AddNamedArgumentsCodeFixProvider.cs" />
```

Comment:
What is editor specific about thsi feature?  I would have thought it would be in the CSharpFeatures library.